helm/kagent/templates/engine-configmap.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
```

Comment:
Can we move these back into the deployment directly, there aren't any secrets in here or anything like that so it's just unecessary obfuscation

Reply:
Ok interesting, I've never done it like this but I buy that. And just to make sure, the deployment will auto roll-out if these values change?